php/wordcountRevB.php

```diff
@@ -15,7 +15,7 @@ function cmp($pairA, $pairB)
 
 $array = array();
 while (false !== ($line = fgets($stdin))) {
-    $words = preg_split('/\s+/', $line);
+    $words = explode(' ', $line);
```

Comment:
also, might not be worthwhile, but *in theory*, `'/ +/'` should work faster, unless of course there are tabs, line endings, etc. i haven't looked at the dataset.